src/BenchmarkDotNet.Core/Helpers/ExternalToolsHelper.cs

```diff
@@ -17,7 +17,7 @@ public static class ExternalToolsHelper
         /// Output of the `cat /proc/info` command.
         /// Linux only.
         /// </summary>
-        public static readonly Lazy<Dictionary<string, string>> ProcCpuInfo = LazyDic(RuntimeInformation.IsLinux, "cat", "/proc/cpuinfo", ':');
+        public static readonly Lazy<ProcCpuInfoParser> ProcCpuInfo = LazyProcCpuInfo(RuntimeInformation.IsLinux, "cat", "/proc/cpuinfo");
```

Comment:
It's a good idea to replace `Dictionary<string, string>` by `ProcCpuInfoParser`!